packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -250,3 +278,26 @@ class TooltipTheme extends InheritedTheme {
   @override
   bool updateShouldNotify(TooltipTheme oldWidget) => data != oldWidget.data;
 }
+
+/// The method of interaction that will trigger a tooltip.
+/// Used in [Tooltip.triggerMode] and [TooltipThemeData.triggerMode].
+enum TooltipTriggerMode {
+  /// Tooltip will only be shown by calling `ensureTooltipVisible`.
+  manual,
+
+  /// Tooltip will be shown after a long press.
+  ///
+  /// See also:
+  ///
+  ///   * [GestureDetector.onLongPress], the event that is used for trigger.
+  ///   * [Feedback.forLongPress], the feedback method called when feedback is enabled.
+  longPress,
+
+  /// Tooltip will be shown after a single tap.
+  ///
+  /// See also:
+  ///
+  ///   * [GestureDetector.onTap], the event that is used for trigger.
+  ///   * [Feedback.onTap], the feedback method called when feedback is enabled.
```

Comment:
This appears to be the cause of the failing docs check.

Reply:
.. can you give more info on the actual reason though?
Without it I can't know if it's something I can fix, and if so how, or if I should just remove it..

Reply:
You can see the output of failing checks under the checks tab by selecting the one that is failing.
In this case,  simply `Feedback.onTap` does not exist.

Reply:
yeah saw that and fixed that before your comment.
I thought the builds were private but I finally managed to find the very little link to the full details and fixed it already.